train_config/training_server/flash.py

```diff
@@ -64,9 +69,10 @@ def main():
         train_config['token'],
         train_config['task_type'],
         accuracy,
-        model_path,
+        model_path_pt,
         acc_plot_path,
         metadata,
+        model_path_onnx=model_path_onnx if model_path_onnx else None
```

Comment:
This will throw an error. Define this variable on top as `None`

Reply:
Resolved in the latest push.